StripeIdentity/StripeIdentity/Source/NativeComponents/ViewControllers/PhoneOtpViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PhoneOtpViewController: IdentityFlowViewController {
+    private let phoneOtpContent: StripeAPI.VerificationPageStaticContentPhoneOtpPage
+
+    let phoneOtpView: PhoneOtpView
+
+    var flowViewModel: IdentityFlowView.ViewModel {
+        return .init(
+            headerViewModel: .init(
+                backgroundColor: .systemBackground,
+                headerType: .plain,
+                titleText: phoneOtpContent.title
+            ),
+            contentView: phoneOtpView,
+            buttons: [
+                .init(
+                    text: phoneOtpContent.resendButtonText,
+                    state: {
+                        switch phoneOtpView.viewModel {
+                        case .InputtingOTP:
+                            return .enabled
+                        case .SubmittingOTP:
+                            return .disabled
+                        case .ErrorOTP:
+                            return .enabled
+                        case .RequestingOTP:
+                            return .loading
+                        case .RequestingCannotVerify:
+                            return .disabled
+                        }
+                    }(),
+                    isPrimary: false,
+                    didTap: {
+                        self.generateOtp()
+                    }
+                ),
+                .init(
+                    text: phoneOtpContent.cannotVerifyButtonText,
+                    state: {
+                        switch phoneOtpView.viewModel {
+                        case .InputtingOTP:
+                            return .enabled
+                        case .SubmittingOTP:
+                            return .disabled
+                        case .ErrorOTP:
+                            return .enabled
+                        case .RequestingOTP:
+                            return .disabled
+                        case .RequestingCannotVerify:
+                            return .loading
+                        }
+                    }(),
+                    isPrimary: false,
+                    didTap: {
+                        self.phoneOtpView.configure(with: .RequestingCannotVerify)
+                        self.sheetController?.sendCannotVerifyPhoneOtpAndTransition { [weak self] in
+                            self?.phoneOtpView.reset()
+                        }
+                    }
+                ),
+            ]
+        )
+    }
+
+    init(
+        phoneOtpContent: StripeAPI.VerificationPageStaticContentPhoneOtpPage,
+        sheetController: VerificationSheetControllerProtocol
+    ) {
+        self.phoneOtpContent = phoneOtpContent
+
+        let bodyText = {
+            if let localPhoneNumber = sheetController.collectedData.phone {
+                return phoneOtpContent.body.replacingOccurrences(of: "&phone_number&", with: localPhoneNumber.phoneNumber!.suffix(4))
```

Comment:
`localPhoneNumber.phoneNumber!` 

I'd always be careful with using `!` because it means it will crash if its null

just making a note to think about it if OK

Reply:
changed to `??` and added comments and provided a default value `""` - this should never happen but would prevent it from crashing